spectrum/autism_prevalence_map/templates/autism_prevalence_map/base.html

```diff
@@ -20,84 +20,55 @@
     <meta property="og:image" content="https://prevalence.spectrumnews.org/static/autism_prevalence_map/img/PrevalenceMapSG.png" />
     <meta property="og:description" content="The map features a collection of studies on autism prevalence around the world. It highlights places where information is available — and places where information is missing." />
 
-    <!-- Mobile Specific Metas
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <!-- CSS libraries
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <!-- <link rel="stylesheet" href="https://use.typekit.net/hjy6pmo.css"> -->
-    <script src="https://use.typekit.net/gnz4dab.js"></script>
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/vendor/css/bootstrap.min.css' %}">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
-    <!-- Custom site-wide CSS
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/css/style.css' %}">
+    {% if environment == 'development' %}
+        <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/dist/main.css' %}">
+    {% else %}
+        <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/dist/main.min.css' %}">
+    {% endif %}
```

Comment:
1. use single quote
2. this is repeating a lot. maybe just put the conditional logic in views.py to send min or not as context_dict, so that this file does not have the logic?

Reply:
Yeah, how about sending the file extension as the `context_dict` since that's what it's for? View files are not supposed to have logic as much as possible.